data/test-data.inc_sample.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * To use:
+ * Copy this file to /data/test-data.inc.php
+ * Make sure the lines related to this file and UNL_Peoplefinder::$sampleUID are uncommented in config.inc.php
+ */
+
+/**
+ * Sample of the result of the query in UNL_Peoplefinder_Driver_OracleDB->fixLDAPEntries()
+ * Simulates the result for one person: hhusker1
+ */
+UNL_Peoplefinder_Driver_OracleDB::$sampleFixLDAPEntries =
+array (
+    0 =>
+        array (
+            'NETID' => 'hhusker1',
+            'NU_FERPA' => NULL,
+            'MAIL' => 'NOREPLY@UNL.EDU',
+        ),
+);
+
+
+/**
+ * Sample of the result of ldap_get_entries().
+ * Simulates a result for one person: hhusker1
+ */
+UNL_Peoplefinder_Driver_LDAP::$samplePersonLDAP =
+array (
+    'count' => 1,
+    0 =>
+        array (
+            'cn' =>
+                array (
+                    'count' => 1,
+                    0 => 'hhusker1',
+                ),
+            0 => 'cn',
+            'sn' =>
+                array (
+                    'count' => 1,
+                    0 => 'Husker',
+                ),
+            1 => 'sn',
+            'title' =>
+                array (
+                    'count' => 1,
+                    0 => 'Mascot',
+                ),
+            2 => 'title',
+            'postaladdress' =>
+                array (
+                    'count' => 1,
+                    0 => 'MSTD, UNL, 685880120',
+                ),
+            3 => 'postaladdress',
+            'telephonenumber' =>
+                array (
+                    'count' => 1,
+                    0 => '4024724224',
+                ),
+            4 => 'telephonenumber',
+            'givenname' =>
+                array (
+                    'count' => 1,
+                    0 => 'Herbie',
+                ),
+            5 => 'givenname',
+            'displayname' =>
+                array (
+                    'count' => 1,
+                    0 => 'Herbie Husker',
+                ),
+            6 => 'displayname',
+            'department' =>
+                array (
+                    'count' => 1,
+                    0 => 'Athletics                            UNL',
```

Comment:
Is the whitespace in string intentional?

Reply:
Yes, that is an example of what is returned by LDAP/AD. There is code in this project that processes those strings and cleans them up for public view.